user-docs/pupil-capture.md

```diff
@@ -209,67 +224,190 @@ If you open up a session folder you will see a collection of video(s) and data f
+ 	'gaze_point_3d': [0.8822507422478054, -18.62344068675104, 510.7932426103372],
+ 	'base_data': [<pupil datum>],  # list of pupil data that was used to calculate the gaze
+ 	'confidence': 1.0,
+ 	'topic': 'gaze'}
 ```
-We have written some simple Python scripts that you can try using Pupil Server to have your gaze control a mouse. Or just print out streaming from Pupil Server. For more simple scripts, check out the [pupil-helpers repository](https://github.com/pupil-labs/pupil-helpers).
 
-### Message Format for Pupil Server
-Messages from pupil server mirror all objects in the `events` dict that is used internally in pupil capture and player. The data is send per topic (`pupil_positions`, `gaze_positions` ...) and serialized using [json](https://docs.python.org/2/library/json.html). The example above tells it all.
+See [Data format](#data-format) for a detailed explanation of the dictionary keys. -->
+
+#### Pupil Time Sync
+
+If you want to record data from multiple sensors (e.g. multiple Pupil Capture instances)
+with different sampling rates it is important to synchronize the clock of each sensor.
+You will not be able to reliably correlate the data without the synchronization.
+
+The [Pupil Time Sync protocol](https://github.com/pupil-labs/pupil/blob/0fbccd412a9e0ff553eb91727dd0da54d33e9637/pupil_src/shared_modules/time_sync_spec.md)
+defines how multiple nodes can find a common clock master and synchronize their time with it.
+
+The Pupil Time Sync plugin is able to act as clock master as well as clock follower.
+This means that each Pupil Capture instance can act as a clock reference for others
+as well as changing its own clock such that it is synchronized with an other reference
+clock.
+
+Pupil Time Sync nodes only synchronize time within their respective group. Be aware
+that each node has to implement the same protocol version to be able to talk to
+each other.
+
+<aside class="notice">
+See the [pupil-helpers](https://github.com/pupil-labs/pupil-helpers/tree/master/pupil_sync) for example Python implementations.
+</aside>
+
+<aside class="notice">
+For this to work your network needs to allow `UDP` transport. If the nodes do not find each other, create a local wifi network and use that instead.
+</aside>
+
+#### Frame Publisher
+<!-- Not sure if this should be mentioned here at all -->
+The `Frame Publisher` plugin broadcasts video frames from the world and eye cameras.
+
+There is a [pupil-helper example script](https://github.com/pupil-labs/pupil-helpers/blob/0df77b47cebd49a6c35b6769da483c115a626836/pupil_remote/recv_world_video_frames.py) that showcases how to receive and decode world frames.
+
+#### Remote Recorder
+
+This plugin is able to remotely start and stop recordings on all connected
+Pupil Mobile sources. This is different to the normal recorder that creates
+a recording on the device running Pupil Capture. The `Remote Recorder` plugin
+creates recordings on the phone.
```

Comment:
lack of context for the last sentence. Pupil Mobile reference needed and link to Pupil Mobile within this page.